src/main/java/org/broadinstitute/hellbender/tools/copynumber/allelic/alleliccount/AllelicCountTableColumn.java

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.hellbender.tools.copynumber.allelic.alleliccount;
+
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+
+/**
+ * Table columns of an allelic count tab-separated file.
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+enum AllelicCountTableColumn {
+    CONTIG,
+    POSITION,
+    REF_COUNT,
+    ALT_COUNT,
+    REF_NUCLEOTIDE,
```

Comment:
you could structure this after `TargetTableColumn` class, i.e. specify which columns are optional